lib/relation-definition.js

```diff
@@ -1247,6 +1286,8 @@ RelationDefinition.belongsTo = function(modelFrom, modelTo, params) {
 
     modelFrom.dataSource.defineProperty(modelFrom.modelName, discriminator, {type: 'string', index: true});
   } else {
+    // relation is not polymorphic
+    modelTo = lookupModelTo(modelFrom, modelToRef, params);
```

Comment:
please keep the check of `'string' === typeof modelToRef` here, see old code https://github.com/strongloop/loopback-datasource-juggler/pull/1301/files#diff-20531c2f86840f4a537a4db9f742b0e2L1220